proselint/tools.py

```diff
@@ -13,6 +13,7 @@
 import sys
 import traceback
 from warnings import showwarning as warn
+from . import config
```

Comment:
```suggestion

from . import config
```
Separating local or external imports from builtins are necessary in lint rules.